pkg/rancher-desktop/backend/k3sHelper.ts

```diff
@@ -678,7 +689,18 @@ export default class K3sHelper extends events.EventEmitter {
 
           sums[filename] = sum;
         }
-        const promises = [this.filenames.exe, this.filenames.images].map(async(filename) => {
+        let existingIndex;
+
+        for (let index = 0; existingIndex === undefined && index < this.filenames.images.length; index++) {
+          if (fs.existsSync(path.join(dir, this.filenames.images[index]))) {
```

Comment:
Ah, it looks like [`fs.exists()`](https://nodejs.org/docs/latest-v16.x/api/fs.html#fsexistspath-callback) is deprecated; presumably, this will need `await fs.promises.access(path.join(…), fs.constants.R_OK)` instead?  Annoyingly, that throws instead of returning false.

Reply:
Testing showed fs.promises.access works great, so I have made the change.